test/functions.php

```diff
@@ -0,0 +1,49 @@
+<?php
+require 'src/functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class Functions extends TestCase
+{
+    public function testDisplayDreamDetails ()
+    {
+        $array = 
+        [
```

Comment:
with array variables it is correct to put the opening bracket on the same line, e.g. `$array = [`

Reply:
done

Reply:
hmmm are you sure? 👀